test/Microsoft.AspNetCore.Server.Kestrel.Performance/runtimeconfig.template.json

```diff
@@ -1,5 +0,0 @@
-{
-  "configProperties": {
-    "System.GC.Server": true
```

Comment:
It's on by default in Microsoft.NET.Sdk.Web. cref https://github.com/aspnet/websdk/pull/110. If you're not using the websdk, you can set `<ServerGarbageCollection>true</ServerGarbageCollection>`